app/models/subnet/ipv6.rb

```diff
@@ -26,6 +26,10 @@ def self.supported_ipam_modes
     [:eui64, :db, :none]
   end
 
+  def self.supports_tftp?
+    false
```

Comment:
Sorry, I still don't understand, TFTP orchestration is independent of DHCP, in fact some people do static network provisioning with Foreman (http://serverfault.com/questions/609770/using-foreman-to-provision-a-vm-with-static-ip for instance). The EFI part caught me completely off guard, why is that necessary?